### PR TITLE
Add MTE-1444 [v118] toolbar menu settings test

### PR DESCRIPTION
**Pull request by dragosb01**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-1444)

## :bulb: Description
Added automation test for toolbar menu settings.

## :pencil: Checklist
You have to check all boxes before merging
- [ ] Filled in the above information (tickets numbers and description of your work)
- [ ] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

File: Client.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -167,7 +167,7 @@
 		2C473BD0209778900008C853 /* DownloadFilesTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2C473BCF209778900008C853 /* DownloadFilesTests.swift */; };
 		2C49854E206173C800893DAE /* photon-colors.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2C49854D206173C800893DAE /* photon-colors.swift */; };
 		2C4A07DC20246EAD0083E320 /* DragAndDropTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2C4A07DB20246EAD0083E320 /* DragAndDropTests.swift */; };
-		2C4B6BF320349EB800A009C2 /* FirstRunTourTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2C4B6BF220349EB800A009C2 /* FirstRunTourTests.swift */; };
+		2C4B6BF320349EB800A009C2 /* OnboardingTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2C4B6BF220349EB800A009C2 /* OnboardingTests.swift */; };
```
Comment (clarmso) [inline Client.xcodeproj/project.pbxproj:170]: Was `FirstRunTourTests` renamed or removed?

File: Client/Application/ImageIdentifiers.swift [hunk 2]
```diff
@@ -66,4 +66,6 @@ public struct ImageIdentifiers {
     public static let breachedWebsite = "Breached Website"
     public static let whatsNew = "whatsnew"
     public static let zoomIn = "menu-ZoomIn"
+    public static let customizeHomepage = "editLarge"
```
Comment (clarmso) [inline Client/Application/ImageIdentifiers.swift:69]: 👍🏼 The iOS developers have told us to add and to use identifiers.

File: Client.xcodeproj/project.pbxproj (hunk 1 above)
Comment (clarmso) [inline Client.xcodeproj/project.pbxproj:170]: Is the file renamed due to TestRail cleanup?

**Review by clarmso — COMMENTED** [review]
There still some references to FirstRunTourTests in the test plans. Please update all the *.xctestplan to reflect the file name update.

**Comment by dragosb01** [comment]
@clarmso the file renaming is a leftover from my previous task where i forgot to rename the file.

**Comment by dragosb01** [comment]
> There still some references to FirstRunTourTests in the test plans. Please update all the *.xctestplan to reflect the file name update.

@clarmso i did a small change on the testplans but i dont any references to the old name. Please let me know if you notice any.

**Comment by dragosb01** [comment]
> This pull request has conflicts when rebasing. Could you fix it @dragosb01? 🙏

@clarmso rebased and its all green.

**Review by clarmso — APPROVED** [review]

**Review by clarmso — APPROVED** [review]

**Pull request merged**